lib/ansible/plugins/lookup/password.py

```diff
@@ -45,6 +45,9 @@
         version_added: "1.4"
         description:
           - A list of names that compose a custom character set in the generated passwords.
+          - Note that this parameter dictates the possible character sets in the resulting password, not the required character sets.
+            If you want to require certain chacter sets, consider using the C(community.general.random_string lookup) plugin -
```

Comment:
```suggestion
          - This parameter defines the possible character sets in the resulting password, not the required character sets.
            If you want to require certain character sets for passwords, you can use the C(community.general.random_string lookup) plugin -
```

Reply:
@aaronk1 Thanks for the contribution. Some minor wording nits for your consideration.